code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -320,70 +267,75 @@
 	path = /obj/item/clothing/under/dress/dress_cap
 	allowed_roles = list("Site Manager")
 
-/datum/gear/uniform/corpdetsuit
-	display_name = "uniform, corporate (Detective)"
-	path = /obj/item/clothing/under/det/corporate
-	allowed_roles = list("Detective","Head of Security")
-
-/datum/gear/uniform/corpsecsuit
-	display_name = "uniform, corporate (Security)"
+/datum/gear/uniform/uniform_security
+	display_name = "uniform, security selection"
 	path = /obj/item/clothing/under/rank/security/corp
-	allowed_roles = list("Security Officer","Head of Security","Warden")
+	allowed_roles = list("Security Officer","Head of Security","Warden", "Detective")
 
-/datum/gear/uniform/corpwarsuit
-	display_name = "uniform, corporate (Warden)"
+/datum/gear/uniform/uniform_security/New()
+	..()
+	var/secunitype = list()
+	secunitype["officer uniform, corporate"] = /obj/item/clothing/under/rank/security/corp
+	secunitype["officer uniform, navy"] = /obj/item/clothing/under/rank/security/navyblue
+	secunitype["officer uniform, hedberg-hammarstrom"] = /obj/item/clothing/under/hedberg
+	secunitype["detective uniform, corporate"] = /obj/item/clothing/under/det/corporate
+	gear_tweaks += new/datum/gear_tweak/path(secunitype)
+
+/datum/gear/uniform/uniform_warden
+	display_name = "uniform, warden selection"
 	path = /obj/item/clothing/under/rank/warden/corp
 	allowed_roles = list("Head of Security","Warden")
 
-/datum/gear/uniform/corphossuit
-	display_name = "uniform, corporate (Head of Security)"
-	path = /obj/item/clothing/under/rank/head_of_security/corp
-	allowed_roles = list("Head of Security")
+/datum/gear/uniform/uniform_warden/New()
+	..()
+	var/warunitype = list()
+	warunitype["warden uniform, corporate"] = /obj/item/clothing/under/rank/warden/corp
+	warunitype["warden uniform, navy"] = /obj/item/clothing/under/rank/warden/navyblue
+	gear_tweaks += new/datum/gear_tweak/path(warunitype)
 
-/datum/gear/uniform/uniform_hop
-	display_name = "uniform, HoP's dress"
-	path = /obj/item/clothing/under/dress/dress_hop
-	allowed_roles = list("Head of Personnel")
+/datum/gear/uniform/uniform_hos
+	display_name = "uniform, head of security selection"
+	path = /obj/item/clothing/under/rank/security/corp
+	allowed_roles = list("Head of Security")
 
-/datum/gear/uniform/uniform_hr
-	display_name = "uniform, HR director (HoP)"
-	path = /obj/item/clothing/under/dress/dress_hr
+/datum/gear/uniform/uniform_hos/New()
+	..()
+	var/hosunitype = list()
+	hosunitype["HoS uniform, corporate"] = /obj/item/clothing/under/rank/head_of_security/corp
+	hosunitype["HoS uniform, navy"] = /obj/item/clothing/under/rank/head_of_security/navyblue
+	hosunitype["detective uniform, corporate"] = /obj/item/clothing/under/det/corporate
```

Comment:
I have a feeling this should be under security selection rather than head of security selection.